packages/pangea-sdk/pyproject.toml

```diff
@@ -1,40 +1,40 @@
-[tool.poetry]
-name = "pangea-sdk"
-version = "3.4.0"
-description = "Pangea API SDK"
-authors = ["Glenn Gallien <glenn.gallien@pangea.cloud>"]
-license = "MIT"
-readme = "README.md"
-homepage = ""
-repository = ""
-keywords = ["Pangea", "SDK", "Audit"]
-classifiers = [
-    "Topic :: Software Development",
-    "Topic :: Software Development :: Libraries",
-]
-packages = [
-    { include = "pangea" }
-]
-
-[tool.poetry.dependencies]
-python = "^3.7.2"
-requests = "^2.31.0"
-cryptography = "41.0.5"
-schema = "^0.7.5"
-python-dateutil = "^2.8.2"
-alive-progress = "^2.4.1"
-pydantic = "^1.10.2"
-pytest = "^7.2.0"
-deprecated = "^1.2.13"
-aiohttp = "^3.8.6"
-asyncio = "^3.4.3"
-google-crc32c = "^1.5.0"
-
-[tool.poetry.dev-dependencies]
-mypy = "^0.941"
-black = "^22.1.0"
-docstring-parser = "^0.14.1"
-
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+[tool.poetry]
+name = "pangea-sdk"
+version = "3.4.0"
+description = "Pangea API SDK"
+authors = ["Glenn Gallien <glenn.gallien@pangea.cloud>"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://pangea.cloud/docs/sdk/python/"
+repository = "https://github.com/pangeacyber/pangea-python/tree/main/packages/pangea-sdk"
```

Comment:
should this just point to 
`https://github.com/pangeacyber/pangea-python`?

Reply:
I considered that but since this is a monorepo (albeit of a single package) it _could_ have multiple packages and I figured it would be more convenient to link to the exact location of each. I'm totally okay with changing this though.

Reply:
I guess since it's pointing to exactly the pyproject, it's less confusing than looking at the toplevel tree. I'm cool with this